Bottom: b17b771cfdd3fa087c28a8eb9d115e4690b5a2c8
Top:    80171986e416410b4f53ee20439f8220d615ae5a
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-14 00:54:52 +0000

Refresh of add-polygon-parentage-tree

---

diff --git a/src/polygon.c b/src/polygon.c
index d045ce2..8360d64 100644
--- a/src/polygon.c
+++ b/src/polygon.c
@@ -104,9 +104,6 @@ dicer output is used for HIDs which cannot render things with holes
 #include <dmalloc.h>
 #endif
 
-/* For getrlimit, setrlimit */
-#include <sys/time.h>
-#include <sys/resource.h>
 
 #undef DEBUG_CIRCSEGS
 
@@ -123,8 +120,6 @@ static double bw_rotate_circle_seg[4];
 void
 polygon_init (void)
 {
-  struct rlimit limit;
-
   double cos_ang = cos (2.0 * M_PI / POLY_CIRC_SEGS_F);
   double sin_ang = sin (2.0 * M_PI / POLY_CIRC_SEGS_F);
 
@@ -133,12 +128,6 @@ polygon_init (void)
 
   bw_rotate_circle_seg[0] =  cos_ang;  bw_rotate_circle_seg[1] =  sin_ang;
   bw_rotate_circle_seg[2] = -sin_ang;  bw_rotate_circle_seg[3] =  cos_ang;
-
-  /* DEBUG - AVOID PCB running the system out of memory! */
-  getrlimit (RLIMIT_AS, &limit);
-  limit.rlim_cur = MIN (limit.rlim_cur, 7000 * 1024 * 1024 /* 2000 GiB limit to virtual memory size */);
-  setrlimit (RLIMIT_AS, &limit);
-
 }
 
 Cardinal
